lib/testers/rule-tester.js

```diff
@@ -490,7 +490,19 @@ RuleTester.prototype = {
                 }
 
                 if (item.hasOwnProperty("output")) {
-                    const fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+                    let fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+
+                    if (item.multipass) {
```

Comment:
I'm a bit confused about why it's necessary to add an option here. Could you clarify?

My concern is that this would be a change to core, which I'd rather avoid until we decide on a general solution to conflicting fixes.